docs/core/entity/calendar.md

```diff
@@ -48,6 +48,9 @@ A calendar entity can return events that occur during a particular time range. S
 - The `end_date` is the upper bound and applied to the event's `start` (exclusive).
 - Recurring events should be flattened and returned as individual `CalendarEvent`.
 
+The start and end datetime will always have a timezone, which the implementation should be use for
```

Comment:
The grammar here sounds weird.

Reply:
Thanks will send another update.

Reply:
I've send #1603 to update this and we can also use it to complete your review of this PR. I explicitly assigned you as a reviewer on this PR originally to make sure we were aligned so don't merge it Paulus :)